rpcs3/Emu/system_config.cpp

```diff
@@ -16,3 +18,14 @@ std::string cfg_root::node_sys::get_random_system_name()
 	std::srand(static_cast<u32>(std::time(nullptr)));
 	return "RPCS3-" + std::to_string(100 + std::rand() % 899);
 }
+
+u128 cfg_root::node_sys::get_random_psid()
+{
+	std::random_device rnd;
+	std::mt19937_64 prng(rnd());
```

Comment:
unused.

Reply:
Actually, because std::random_device::operator() returns unsigned int, call it 4 times for each 32 bits in the 128 bits value.